dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -0,0 +1,79 @@
+[[expanding_persistent_volumes]]
+= Expanding Persistent Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[overview]]
+== Overview
+
+You can increase the size of the persistent volumes (PVs) that your pods are using. To do this, you can update the persistent volume claim (PVC) to request a new size. The  controller then applies the change to the PV, which is bound to the PVC.
+
+For volume types that require both file system expansion and a volume plugin-based modification, growing PVs is a two-step process. For volume types that only require a volume plugin-based API call, growing PVs is a one-step process.
+
+A new controller called `volume_expand_controller` listens for PVC size expansion requests and takes action as needed. The new controller does the following:
+
+*  Watches for PVC update requests and adds the PVC to the controller's work queue if an increase in volume size was requested. Once the PVC is added to the controller's work queue, `- pvc.Status.Conditions` is updated to `ResizeStarted: True`.
+* For unbound or pending PVCs,` - resize ` triggers no action in `volume_expand_controller`.
+* If `pv.Spec.Capacity already` is greater or equal than the requested size, then no action is performed by the controller.
+* A separate goroutine reads the work queue and performs a corresponding volume resize operation. If there is a resize operation in progress for the same volume, then the resize request is pending and retries once the previous resize request is complete.
+* Controller resizing is level-based rather than edge-based. If there is more than one pending resize request for the same PVC, then new resize requests for the same PVC replace older pending requests.
+* Resize is performed via the volume plugin interface, which is executed inside a goroutine spawned by `operation_exectutor`.
+* A new plugin interface called `volume.Expander` is added to the volume plugin interface. The expander interface defines if the volume requires a file system resize.
+* The controller calls to expand the PVC.
+* Once the volume expansion is successful, the volume is marked as expanded and the new size is reflected or updated in `pv.spec.capacity`. Any errors are reported as events on the PVC object.
+* If the resize operation fails, then `pvc.Status.Conditions` is updated with `ResizeFailed: True`. The corresponding error is also added to the condition field.
+* Depending on the volume type:
```

Comment:
Should the content under "Depending on the volume type" be taken out of the nested list format? I guess it just depends on how it looks when built.

Reply:
Would only mention the user visible parts here, not controller.. would also re-phrase to remove shop talk.